modules/apps/batch-planner/batch-planner-web/src/main/resources/META-INF/resources/js/constants.js

```diff
@@ -60,6 +61,10 @@ export const IMPORT_STRATEGY_SELECTED_EVENT = 'ie-import-strategy-selected';
 export const JSON_FORMAT = 'json';
 export const JSONL_FORMAT = 'jsonl';
 export const NULL_TEMPLATE_VALUE = '';
+export const OBJECT_ENTRY_ENTITY_NAME_PREFIX =
```

Comment:
Hey @sergiojimcos can we use a similar approach to what @jaimelr10 did for the forbidden fields? I think we should not use anything objects specific in the code, instead have it all centralized.

Reply:
Let's fix the root issue: https://github.com/liferay-headless/liferay-portal/pull/1779#issuecomment-1869467606